### PR TITLE
Fix track  backend API

### PR DESCRIPTION
**Pull request by Delaunay**

### CODE REVIEW

File: src/orion/storage/track.py [hunk 1]
```diff
@@ -393,6 +407,8 @@ def fetch_experiments(self, query, selection=None):
         for k, v in query.items():
             if k == 'name':
                 new_query['metadata.name'] = v
+                # FIXME
```
Comment (bouthilx) [inline src/orion/storage/track.py]: Is it fixed? What is to be fixed? :)

Comment (Delaunay) [reply src/orion/storage/track.py]: it is ugly

File: src/orion/storage/track.py [hunk 2]
```diff
@@ -443,7 +459,7 @@ def register_trial(self, trial):
         for p in trial.results:
             metrics[p.name] = [p.value]
 
-        self.current_trial = self.backend.new_trial(TrackTrial(
```
Comment (bouthilx) [inline src/orion/storage/track.py]: No more current trial?

Comment (Delaunay) [reply src/orion/storage/track.py]: it was not used anyway

**Pull request merged**